### DISCUSSION TITLE
Instruction HttpOnly

### DISCUSSION THREAD

**Discussion by blemoineVIVA** [AmauriC/tarteaucitron.js#1282] [Q&A] [answered]
Bonjour,

Un client est revenu vers moi avec un audit de sécurité demandant à ce que tous les cookies soient Secure et HttpOnly.
J'ai vu que Secure est automatiquement ajouté si on est en https, mais je n'ai rien vu vis à vis de HttpOnly.
Est-ce qu'il y a un moyen d'ajouter le HttpOnly sans passé par du paramétrage IIS ou équivalent ?

Ca me fait me demander si tarteaucitron est capable de fonctionner en HttpOnly ?

Merci du temps que vous me consacrez

-----------------------------------------------------------------------
Hello,

A client pushed me the result of his security audit and asked all cookies to be Secure and HttpOnly.
As "Secure" is automaticaly added if protocol is https, can HttpOnly be added in a similar way or thanks to a parameter of the init ?

In insight can tarteaucitron works with HttpOnly ?

Thank you to take the time

**Comment by AmauriC** [comment, marked as answer]
Bonjour,

tarteaucitron étant une librairie Javascript, il faut que le cookie tarteaucitron soit accessible par le navigateur.

Le passer en httponly rendrait la lecture du consentement impossible.
